frontend/lib/app.dart

```diff
@@ -37,7 +38,11 @@ final GoRouter router = GoRouter(
         GoRoute(
           path: '$activationRouteName/:$activationRouteCodeParamName',
           builder: (BuildContext context, GoRouterState state) {
-            return DeepLinkActivation(base64qrcode: Uri.decodeFull(state.uri.fragment));
+            // GoRouter >= 13.x did not handle uri fragments properly without trailing slash. After the package update to recent version this was fixed.
+            // Though we need legacy support for old deep links that have a trailing slash.
+            // Implemented in April 2025, can be removed around April 2027.
```

Comment:
🙃 
```suggestion
            // GoRouter 13.x did not handle uri fragments properly without trailing slash.
            // Therefore we need legacy support for old deep links that have a trailing slash.
            // Implemented in April 2025, can be removed around April 2027.
```

Reply:
Just for clarification: Why does a trailing `/` "break the activation" now? Is it interpreted as part of the base64 encoded code?

Reply:
yes thats the point. before the go router update it wasn't interpreted but needed for whatever reason and now its interpreted as part of the base64 string

Reply:
Perhaps it makes more sense then to strip the slash where the code is interpreted?

Reply:
Hm but we should strip it before decodingUri.
Why do you think that?

Reply:
Since in general trailing slashes are not a problem, even with the new go router upgrade and could perhaps exist for different reasons than just deeplinks. By moving the stripping to where the code is interpreted (before decodeUri) we put the solution/workaround more closely to the problem and avoid problems possibly occurring somewhere in between. Furthermore, this reduces the risk that in the future its not clear anymore why this is needed/just removed imo.

Reply:
Ok so you think i should just send the `state.uri.fragment` without removing the slash and decoding it to the deep link activation and handle it there?

Reply:
Exactly 👍